pkg/collector/python/containers.go

```diff
@@ -23,19 +23,23 @@ import "C"
 var filter *containers.Filter
 
 // IsContainerExcluded returns whether a container should be excluded,
-// based on it's name and image name. Exclusion patterns are configured
+// based on it's name, image name and namespace. Exclusion patterns are configured
 // via the global options (ac_include/ac_exclude/exclude_pause_container)
 //export IsContainerExcluded
-func IsContainerExcluded(name, image *C.char) C.int {
+func IsContainerExcluded(name, image, namespace *C.char) C.int {
```

Comment:
Although this function is quite simple, I think it's worth adding a test for it now (in particular a non-regression test to make sure it will always handle a `nil` namespace gracefully)

(note: adding tests for these functions is a bit weird, it requires one level of indirection, see kubeutil_test.go for example)

Reply:
Test added.
Thanks for your review !